tweak: M90 instead riot shotguns in ERT armory

Mahimka42 opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Меняет служебные дробовики на М90 в оружейной ЕРТ
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/1269912125606723594/1271068409404063866
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
![image](https://github.com/user-attachments/assets/5a6d2151-3eb7-4700-b37d-37a779b58911)

## Тесты
Все работает.
<!-- Как вы тестировали свой код. Ревьюеру будет проще, если он будет знать какие действия вы совершали, проверяя свой ПР. -->

Mahimka42 committed:
Update CentComm.dmm

Daeberdir reviewed (changes requested):
![image](https://github.com/user-attachments/assets/8f05a9ad-76c1-40e0-9362-e6ca9f7b9580)
Раскидай по икс-игреку.

Mahimka42 committed:
Update CentComm.dmm

Daeberdir committed:
Brigadier's tweaks.

- ППшкам возвращено их уникальное название, убирающее прямую аналогию с вооружением Синдиката.
- К 5 единицам огнестрела 10 магазинов. Стандартизация.
- У шаттеров и кнопки к ППшкам установлено актуальное айди и название.
- ППшки визуально лучше раскиданы по икс-игреку.
- Убрано пару лишних декалей, лишние тайлы и зоны из космоса.

Daeberdir reviewed (dismissed):
![imag312412e](https://github.com/user-attachments/assets/556dcc10-7fa0-4e97-8eb5-1dfd15f818ae)

Daeberdir committed:
Addictional tweak.

- Во всей оружейке ЕРТ на стойках для пушек всё правильно раскидано по икс-игреку.

Daeberdir reviewed (approved):
От так лучше.

Den109G reviewed (approved):
Одобряю со стороны таяры2.
